hw_diag/tasks.py

```diff
@@ -53,6 +54,8 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
         value = get_helium_blockchain_height()
     except KeyError as e:
         logging.warning(e)
+    except (ConnectTimeout, ReadTimeout) as e:
+        logging.exception(e)
```

Comment:
`logging.exception` will automatically show the stacktrace. So would be more useful to do something like `logging.exception("Request to Helium API timed out. Will fallback to block height of 1.")`.

Reply:
This makes sense, that way it'll give us a more detailed message of what is going on. I'll add that in